playground/worker/tsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+	"extends": ["@vite-plugin-cloudflare/typescript-config/worker.json"],
+	"include": ["**/*"],
+	"exclude": ["**/__tests__/**/*"]
```

Comment:
This is the only playground that has an `exclude` property in the tsconfig. As you always just extend the playground tsconfig in the `__tests__` tsconfig, it doesn't seem both are necessary.

Reply:
I've been playing around with trying to `exclude` the `__tests__` files and try to get them to pick up the parent tsconfig file in the playground, but that didn't work, I did miss this when cleaning that experimentation up, thanks for noticing it! 🙏